.cspell.json

```
@@ -347,6 +347,7 @@
     "behaviortree",
     "behaviour",
     "behaviours",
+    "bggr",
```

@miursh Can I use the all-capitalized style or should it be small letters? (Same for `grbg`, `grbg`, and `rggb`.)

```suggestion
    "BGGR",
```

---

@kenji-miyake 
Yes,.. I was wondering which way I should do.
The reason those words are in lowercase is because they are defined that way at [here](https://github.com/tier4/spinnaker/blob/main/spinnaker_camera_driver/src/camera_settings.cpp#L30-L36). Therefore, the [config files](https://github.com/tier4/autoware_individual_params.jpntaxi/blob/main/individual_params/config/default/aip_xx1/flir/bfs0.param.yaml#L13-L14) need to follow that way.
Is it a not good to add both all capital letter words and all small letter words to config?
Or, should I add spell check that only affects to the config repository?

---

@miursh Thank you for the comment. Actually, since this dictionary is case-insensitive, either way is fine.
My preference is using uppercase so that we can understand they are not words.

> Is it a not good to add both all capital letter words and all small letter words to config?

So, it's not good because it will be redundant.

> Or, should I add spell check that only affects to the config repository?

I'm sorry, I don't understand this well. :thinking:

---

Oh, I see.
I will fix to capital letters

---

https://github.com/tier4/autoware-spell-check-dict/pull/497/commits/95654345e568c0c054bf2f3b2bc61e098be57863